Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/StaticCommonTraitsPlugin.cs

```diff
@@ -7,17 +7,26 @@ namespace DCL.PerformanceAndDiagnostics.Analytics
     public class StaticCommonTraitsPlugin : EventPlugin
     {
         private readonly string dclRendererType = SystemInfo.deviceType.ToString(); // Desktop, Console, Handeheld (Mobile), Unknown
-        private readonly string sessionID = SystemInfo.deviceUniqueIdentifier + DateTime.Now.ToString("yyyyMMddHHmmssfff");
         private readonly string rendererVersion = Application.version;
         private readonly string runtime = Application.isEditor? "editor" : Debug.isDebugBuild ? "debug" : "release";
         private readonly string os = SystemInfo.operatingSystem;
 
+        private readonly string sessionId;
+        private readonly string launcherAnonymousId;
+
         public override PluginType Type => PluginType.Enrichment;
 
+        public StaticCommonTraitsPlugin(string launcherAnonymousId, string sessionId)
+        {
+            this.sessionId = !string.IsNullOrEmpty(this.sessionId) ? sessionId : SystemInfo.deviceUniqueIdentifier + DateTime.Now.ToString("yyyyMMddHHmmssfff");
```

Comment:
Is it a valid case when sessionId could be null?

Reply:
yes, in case if I will not get it from the Launcher. For example if I playtest simple build or from the Editor

Reply:
however, good catch. Autocompletion put `this.` in a wrong place 😅